### PR TITLE
[T4A4][F12-A2] Aditya Hendraatmaja

### PR DESCRIPTION
**Pull request by ahendraatmaja**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Returns a concatenated version of the printable strings of each object.
+ */
+public interface Printable {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Printable.java:6]: Glad to see header comments =).

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	public static final String PRINTABLE_SEPARATOR = ", ";
+	
+	public String getPrintableString(); 
+	
+	public static String getPrintableString(Printable...printables) {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Printable.java:11]: Do recall from tutorial on commenting for methods.

File: src/seedu/addressbook/data/person/Printable.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+		String stringOutput = "";
+		for (Printable printable : printables) {
+			stringOutput = stringOutput + printable + PRINTABLE_SEPARATOR;
+		}
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/data/person/Printable.java:15]: Perhaps you can use `StringJoiner` to concatenate your string?

**Review by brandonyeoxg — COMMENTED** [review]
@ahendraatmaja Some comments added. Please close the PR after reading comments.

File: src/seedu/addressbook/data/person/Printable.java (hunk 3 above)
Comment (ahendraatmaja) [reply src/seedu/addressbook/data/person/Printable.java:15]: Thanks for the input! I wasn't aware of this feature in Java :)

**Pull request closed without merging**